src/fty_proto.xml

```diff
@@ -73,39 +84,71 @@
     id = "2"
     >
     <field name = "rule" type = "string">
-        a rule name, that triggers this alert
+        Rule name that triggers this alert.
     </field>
-    <field name = "element_src" type = "string">
-        name of the element, where alert was detected. Most of the time is would be asset element name
+
+    <field name = "name" type = "string">
+        Name of asset where alert is trigged.
     </field>
+
     <field name = "state" type = "string">
-        state of the alert. Possible values are ACTIVE/ACK-WIP/ACK-IGNORE/ACK-PAUSE/ACK-SILENCE/RESOLVED
+        Alert state.
+
+        Permissible values:
+        ACTIVE / ACK-WIP / ACK-IGNORE / ACK-PAUSE / ACK-SILENCE / RESOLVED
     </field>
+
     <field name = "severity" type = "string">
-        severity of the alert. Possible values are INFO/WARNING/CRITICAL
+        Alert severity.
+
+        Permissible values:
+        INFO / WARNING / CRITICAL
     </field>
+
     <field name = "description" type = "string">
-        a description of the alert
+        Alert description.
     </field>
+
     <field name = "time" type = "number" size = "8">
-        ALERT date/time
+        Alert timestamp - unixtime.
     </field>
+
     <field name = "action" type = "string">
-        list of strings separated by "/" ( EMAIL/SMS ) ( is optional and can be empty )
+        List of slash-separated ('/') actions, e.g.: "EMAIL/SMS".
+        Can be empty.
+    </field>
+
+    <field name = "ttl" type = "number" size = "4">
+        Time to live in seconds.
+
+        Value defines duration of validity (i.e. how long is alert valid, when
+        it expires).
+        When current unixtime is greater than time + ttl this alert expired
+        and is no longer valid.
     </field>
 </message>
 
 <message
     name = "ASSET"
     id = "3">
     <field name = "name" type = "string">
-        Unique name of asset.
+        Asset name.
     </field>
+
     <field name = "operation" type = "string">
-        What have hapened with asset (create|update|delete|inventory).
+        Asset operation.
+
+        Permissible values:
+        create / update / delete / inventory / retire
+
+        A little explanation. Due to historical reasons:
+        * 'update' is actually 'delete' + 'create'
```

Comment:
'update' is actually 'delete' + 'create', so only data passed in this message becomes the new content of asset information.